metricbeat/module/apache/_beat/config.yml

```diff
@@ -1,4 +1,4 @@
-# Apache Module
+#----------------------------- Apache Module ----------------------------------
```

Comment:
I hope to auto generate this line in the future based on a name entry in fields.yml.